ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/defendant/ResponseAssert.java

```diff
@@ -142,6 +146,8 @@ private void assertFullDefenceResponse(CCDRespondent respondent) {
         fullDefenceResponse.getTimeline().ifPresent(assertDefendantTimelineConsumer(respondent));
 
         fullDefenceResponse.getEvidence().ifPresent(assertDefendantEvidenceConsumer(respondent));
+
+        fullDefenceResponse.getDirectionsQuestionnaire().ifPresent(Assertions::assertThat);
```

Comment:
can we please verify that if its not present in domain object then in CCD it's null as well?